modules/core/src/main/scala/org/scalasteward/core/buildtool/sbt/package.scala

```diff
@@ -35,10 +35,22 @@ package object sbt {
       Dependency(sbtGroupId, sbtArtifactId, sbtVersion.value)
     }
 
-  val scalaStewardScalafixSbt: FileData =
+  val sbtScalaFixDependency: Scope[Dependency] =
+    Scope(
+      Dependency(
+        GroupId("ch.epfl.scala"),
+        ArtifactId("sbt-scalafix"),
+        "",
+        Some(SbtVersion("1.0")),
+        Some(ScalaVersion("2.12"))
+      ),
+      List(Resolver.mavenCentral)
+    )
+
+  def scalaStewardScalafixSbt(version: String): FileData =
     FileData(
       "scala-steward-scalafix.sbt",
-      """addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.34")"""
+      s"""addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "$version")"""
```

Comment:
We can now also remove the plugin from the build:
https://github.com/scala-steward-org/scala-steward/blob/b0699bf8b234bfbdb6444c8d4e5558d28b062261/project/plugins.sbt#L16-L17